SU-SWS/stanford_wysiwyg
# SITES-833: Update wysiwyg feature to support > 2.5 of WYSIWYG

# READY FOR REVIEW

# Summary
- Updates to the new format of feature export for WYSIWYG.

# Needed By (Date)
- Soonish.

# Urgency
- 6/10

# Steps to Test

1. Install a fresh stanford profile install with the default branch and wysiwyg 2.5 
2. Navigate to /admin/config/content/wysiwyg/profile/content_editor_text_format/edit
3. See that no css classes are available
4. Check out this branch
5. Re-install
6. Navigate back to the edit page and validate that all the settings are in place
7. Profit.

# Affected Projects or Products
- All self-serve.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
